### PR TITLE
Euler's Totient Function Editorial - Divisibility Module

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/4_Gold/Divisibility.mdx [hunk 1]
```diff
@@ -507,6 +507,167 @@ $$
 \gcd(a_1, a_2, a_3, a_4) = \gcd(a_1, \gcd(a_2, \gcd(a_3, a_4))).
 $$
 
+## Euler's Totient Function
+
+<FocusProblem problem="gcdex" />
```
Comment (SansPapyrus683) [inline content/4_Gold/Divisibility.mdx:512]: maybe move this down to above the solution?

Comment (brebenelmihnea) [reply content/4_Gold/Divisibility.mdx:512]: Go ahead, do what you want

Comment (brebenelmihnea) [reply content/4_Gold/Divisibility.mdx]: Ok I moved the problem down

**Comment by SansPapyrus683** [comment]
yeah i'll get around to this sometime, just please hold tight!

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by bqi343** [comment]
@brebenelmihnea @SansPapyrus683 @ryanchou-dev This solution TLEs when I tried submitting it.

ref: https://github.com/cpinitiative/usaco-guide/issues/5486

**Pull request merged**